packages/xs-extra/xenctrl.master/opam

```diff
@@ -8,18 +8,13 @@ bug-reports: "https://github.com/xapi-project/xenctrl/issues"
 depends: [
   "ocaml"
   "dune" {build}
+  "conf-m4"
+  "conf-xen"
   "base-unix"
 ]
 build: ["dune" "build" "-p" name "-j" jobs]
 depexts: [
-  ["m4" "libxen-dev" "libsystemd-dev"] {os-distribution = "debian"}
-  ["libxen-dev" "uuid-dev"] {os-distribution = "ubuntu"}
-  ["xen-devel" "libuuid-devel"] {os-distribution = "centos"}
-  ["xen-devel"] {os-distribution = "fedora"}
-  ["xen-devel"] {os-distribution = "rhel"}
-  ["xen-devel"] {os-distribution = "oraclelinux"}
-  ["xen-dom0-libs-devel" "xen-libs-devel"] {os = "xenserver"}
-  ["xen-dev"] {os-distribution = "alpine"}
+  ["libsystemd-dev"] {os-distribution = "debian"}
```

Comment:
Does it really need the systemd headers? This seems more appropiate for the systemd bindings in the transitional libs.

I also removed the libuuid headers dependency here. They might be used elsewhere but not here.

Reply:
xenstored needs systemd, but not xenctrl